tika-core/pom.xml

```diff
@@ -84,6 +84,12 @@
       <artifactId>junit</artifactId>
       <scope>test</scope>
     </dependency>
+      <dependency>
```

Comment:
Please push this into the `tika-translate` module

Reply:
do you mean tika-transcribe?